src/Tile/_ProductTileContent.js

```diff
@@ -1,37 +1,10 @@
-import classnames from 'classnames';
```

Comment:
I think this sub-component can be removed altogether as the only thing it does is pass a `productTile` prop to `TileContent`.  This can be done by cloning children with the `Tile` component and passing the top-level `productTile` prop.